python/docs/examples.rst

```diff
@@ -219,6 +219,26 @@ Using Pandas DataFrames
     # clear the storage and delete the pipeline
     pipeline.delete(True)
 
+Using Completion Tokens
```

Comment:
I don't think it's necessary to document how completion tokens are used internally. I think issue #4819 was about exposing completion tokens via the SDK and documenting their use.

Exposing completion tokens explicitly can be useful with other connector types where they cannot be hidden inside the API.